src/strategy_manager/strategy_manager.hpp

```diff
@@ -0,0 +1,36 @@
+/******************************************************************************
+ * Copyright © 2013-2019 The Komodo Platform Developers.                      *
+ *                                                                            *
+ * See the AUTHORS, DEVELOPER-AGREEMENT and LICENSE files at                  *
+ * the top-level directory of this distribution for the individual copyright  *
+ * holder information and the developer policies on copyright and licensing.  *
+ *                                                                            *
+ * Unless otherwise agreed in a custom licensing agreement, no part of the    *
+ * Komodo Platform software, including this file may be copied, modified,     *
+ * propagated or distributed except according to the terms contained in the   *
+ * LICENSE file                                                               *
+ *                                                                            *
+ * Removal or modification of this copyright notice is prohibited.            *
+ *                                                                            *
+ ******************************************************************************/
+
+#include <vector>
+#include <unordered_map>
+
+namespace antara {
+  struct asset {
+    std::string symbol;
```

Comment:
for symbol we probably want an st_symbol with std::string as underlying type

Reply:
It should really be:
```
Pair { Asset, Asset }
Asset { Symbol }
```
I've made an Asset type, and st_symbol. It's best to make this change now otherwise it will be a pain to change later. I've fixed coinpaprika and I'm doing the same to its tests now.

Reply:
Asset represents a particular coin. We'll need this type for when we cover things like deposits, withdrawals, holdings.

Reply:
Just sync with master st type library is fixed now

Reply:
Some changes have been processed yesterday evening